### PR TITLE
Add duplicate posting reply

### PR DESCRIPTION
**Pull request by saade**
➿  its the `:loop:` emoji.

### CODE REVIEW

**Comment by zepfietje** [comment]
Other emoji suggestion: 🔁.

**Comment by saade** [comment]
oops sorry

File: index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ client.on('messageReactionAdd', async (reaction) => {
         '🥴': `Please read the <#${process.env.GUIDELINES_CHANNEL_ID}> about how to format your code properly.`,
         '✋': `Your question is posted in the wrong channel. Each package and plugin has its own channel.`,
         '⬆️': `Please upgrade to the latest Filament version using \`composer update\`.`,
+        '➿': `Please do not post the same question across different channels.`,
```
Comment (zepfietje) [inline index.js:24]:
```suggestion
        '🔀': `Please don't send the same message in different channels.`,
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ client.on('messageReactionAdd', async (reaction) => {
         '🥴': `Please read the <#${process.env.GUIDELINES_CHANNEL_ID}> about how to format your code properly.`,
         '✋': `Your question is posted in the wrong channel. Each package and plugin has its own channel.`,
         '⬆️': `Please upgrade to the latest Filament version using \`composer update\`.`,
+        '🔀': `Please don't send the same message in different channels.`,
```
Comment (zepfietje) [inline index.js:24]:
```suggestion
        '🔀': `Please don't post the same message across different channels.`,
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

**Pull request merged**